src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -514,48 +514,59 @@ public void TestExecuteNonQuery(string connection, bool isAsync)
 
             Parallel.For(0, 10, i =>
             {
-                IList<object> values = GetValues(dataHint: 45 + i + 1);
-                int numberOfRows = 10 + i;
-
-                // Insert a bunch of rows in to the table.
-                int rowsAffected = InsertRows(tableName: tableName, numberofRows: numberOfRows, values: values, connection: connection);
-
-                Assert.Equal(numberOfRows, rowsAffected);
-                rowsAffected = -1;
-                using (SqlConnection sqlConnection = new SqlConnection(connection))
+                bool tryagain = false;
+                do
                 {
-                    sqlConnection.Open();
+                    IList<object> values = GetValues(dataHint: 45 + i + 1);
+                    int numberOfRows = 10 + i;
 
-                    // Update the set of rows that were inserted just now. And verify the rows affected as returned by ExecuteNonQuery.
-                    using (SqlCommand sqlCommand = new SqlCommand(
-                        cmdText: $"UPDATE [{tableName}] SET FirstName = @FirstName WHERE CustomerId = @CustomerId",
-                        connection: sqlConnection,
-                        transaction: null,
-                        columnEncryptionSetting: SqlCommandColumnEncryptionSetting.Enabled))
+                    // Insert a bunch of rows in to the table.
+                    int rowsAffected = InsertRows(tableName: tableName, numberofRows: numberOfRows, values: values, connection: connection);
+
+                    Assert.Equal(numberOfRows, rowsAffected);
+                    rowsAffected = -1;
+                    using (SqlConnection sqlConnection = new SqlConnection(connection))
                     {
-                        sqlCommand.CommandTimeout = 90;
-                        if (DataTestUtility.EnclaveEnabled)
+                        try
                         {
-                            //Increase Time out for enclave-enabled server.
-                            sqlCommand.CommandTimeout = 90;
-                        }
+                            sqlConnection.Open();
+
+                            // Update the set of rows that were inserted just now. And verify the rows affected as returned by ExecuteNonQuery.
+                            using (SqlCommand sqlCommand = new SqlCommand(
+                                cmdText: $"UPDATE [{tableName}] SET FirstName = @FirstName WHERE CustomerId = @CustomerId",
+                                connection: sqlConnection,
+                                transaction: null,
+                                columnEncryptionSetting: SqlCommandColumnEncryptionSetting.Enabled))
+                            {
+                                sqlCommand.CommandTimeout = 90;
```

Comment:
We only need to increase the CommandTimeout when enclave is enabled.

Reply:
I think that was already there, I just removed the additional check after that which was unnecessary.

Reply:
It is introduced in this [PR](https://github.com/dotnet/SqlClient/commit/1e8e3b85d777e192c2b015195f7f13bee1ab9702#diff-ac18e1392e0d1df72296c274cdf576f1) recently by mistake.

Reply:
Ok, lemme fix that everywhere then.

Reply:
And since you are working on this file, can you also add enclave check for `public void TestSqlDataAdapterFillDataTable(string connection)` Line 421 ? We don't need to increase the timeout for the regular server I think.